RPIPs/RPIP-4.md

```diff
@@ -14,65 +14,68 @@ The Rocket Pool community at large agrees to use Snapshot voting for governance
 The Snapshot voting mechanism uses staked and weighted RPL token voting.
 
 ## Motivation
-Governance in Rocket Pool is currently ad hoc, and there is no way for the community to adopt proposals. We must adopt a governance mechanism, even if imperfect, before any governance actions can be taken legitimately, such as but not limited to spending pDAO funds.
+Governance in Rocket Pool is currently ad hoc, and there is no way for the community to adopt proposals. We must adopt a governance mechanism, even if imperfect, before any governance actions can be taken legitimately, such as spending pDAO funds.
 
 ## Implementation
   
-All Rocket Pool governance proposals must first be posted as a new topic on the governance forums at dao.rocketpool.net. Topics should include a poll with the following format:
+All Rocket Pool governance proposals MUST have an associated post on the governance forums at dao.rocketpool.net.
+Promising community sentiment and a corresponding RPIP document are REQUIRED for snapshot voting to be scheduled.
+
+Topics SHOULD approximate community sentiment by including a poll with the following format which MUST be live for at least [7] days:
 
 ```
 [poll type=regular results=always chartType=bar]
 * Support
 * Oppose
+* Third option
+* etc
 [/poll]
 ```
 
-A favorable forum poll which is live for at least [7] days and a corresponding RPIP document are necessary for snapshot voting to be scheduled. Once the requirements have been met, one of the following parties will create a snapshot vote and publicize it:
-
-- Darren Langley AKA langers (General Manager - Rocket Pool Pty Ltd)
-- Mike Leach AKA Wander (RPIP Editor, Founder - Bamboo Financial Technologies)
-- [others?]
+Once the requirements have been met, either 1) Darren Langley AKA langers (General Manager - Rocket Pool Pty Ltd) or 2) a party he entrusts with this ability MUST create a snapshot vote and publicize it as soon as reasonably possible unless exercizing their veto power (see Vetoing below). 
 
-The snapshot vote will run for [14] days and requires a [50.1%] majority to be successful.
+The snapshot vote will run for [14] days and requires a [15%] voting power quorum to be successful. In the case where no quorum has been met for three of the last four polls, the needed quorum will be reduced by [25%] permanently unless governance action dictates otherwise.
```

Comment:
```suggestion
The snapshot vote SHALL run for [14] days and SHALL REQUIRE a [15%] voting power quorum to be successful. In the case where no quorum has been met for three of the last four polls, the needed quorum SHALL be reduced by [25%] permanently unless governance action dictates otherwise.
```

Reply:
Not sure if the vote duration is really a SHALL (required and inflexible) or SHOULD (recommended). Maybe even "SHALL run for at least [7] days, with [14] days being RECOMMENDED"?